analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.cs

```diff
@@ -176,6 +175,51 @@ public void Execute_PersistSymbols_InsideBlock()
                 ((ILocalReferenceOperation)operation.Instance).Local;
         }
 
+        [TestMethod]
+        public void Execute_UnusedVariable_ClearedAfterBlock()
+        {
+            const string code = @"
+var first = boolParameter ? true : false;
+Tag(""BeforeLastUse""); 
+bool second = first;
+if(boolParameter)
+    boolParameter.ToString();
+Tag(""AfterLastUse"");
+";
+            ISymbol firstSymbol = null;
+            var postProcess = new PostProcessTestCheck(x =>
+            {
+                if (x.Operation.Instance.TrackedSymbol() is { } symbol)
+                {
+                    if (symbol.Name == "first")
+                    {
+                        firstSymbol = symbol;
+                    }
+                }
```

Comment:
Can be single line now
```suggestion
                firstSymbol ??= x.Operation.Instance.TrackedSymbol() is { Name: "first" } symbol ? symbol : null;
```